### DISCUSSION TITLE
Hi everyone ;)

### DISCUSSION THREAD

**Discussion by aweers** [OpenMined/courses#93] [Introduce Yourself]
Hi everyone, 
I am Alex from Germany. I have some experience with machine learning and planning to deepen my knowledge in this area with a master degree probably starting this summer. 
Thanks for the courses, I just started with the first one and I am very excited about it! 🎉

**Comment by yemikifouly** [comment]
Hello Alex! 👋 Good luck with graduate school and welcome to the discussion board! We hope you enjoy the first course! 🙂
